amr-wind/incflo.H

```diff
@@ -96,8 +96,8 @@ public:
     bool regrid_and_update();
     void pre_advance_stage1();
     void pre_advance_stage2();
-    void do_advance();
-    void advance();
+    void do_advance(const int ifixed_point_iteration);
```

Comment:
I think we can just do `fixed_point_iteration` (remove the `i`, it's not really a convention in amr-wind)